pages.fr/linux/pacman-key.md

```diff
@@ -0,0 +1,36 @@
+# pacman-key
+
+> Script enrobeur pour GnuPG utilisé pour gérer le trousseau de clés de pacman.
+> Plus d'informations : <https://man.archlinux.org/man/community/man-pages-fr/pacman-key.8.fr>.
```

Comment:
We have to use the same link for all translation. That's currently a little limitation of our link tracking system.
```suggestion
> Plus d'informations : <https://man.archlinux.org/man/pacman-key>.
```